### DISCUSSION TITLE
How to use in vitro dissolution models in PK-Sim?

### DISCUSSION THREAD

**Discussion by Prajwala1921** [Open-Systems-Pharmacology/Forum#1217] [How-Tos]
Hello all,

In PK-Sim, dissolution functions like lint 80 are suitable for fast dissolving drugs and the table option can be well utilized for plugging in the specific dissolution data. But, when I am giving a dissolution profile with a peculiar shape it needs to be fitted with the Weibull function but I see no option there to input my experimental dissolution profile to match with the Weibull function. Again if I used only the shape factor to define my profile it is tedious and also I think it cannot reproduce the same experimental profile.
![{999956F6-66E6-46B7-92A5-5DECFB4A605A} png](https://user-images.githubusercontent.com/105360691/170917519-08352a86-0a21-4601-a4c5-9110c51c04d5.jpg)

But these functions are extremely helpful when we are looking for a target dissolution profile for modified release formulation. The model very well translated the predictions for modified release dosage forms. I am only concerned about the dosage forms with a complex release mechanism.
So, my ask here is, are there any other ways of plugging in the experimentally obtained profile in the Weibull function in the formulation tab? 

Thank you in advance!

**Comment by SRuedesh** [comment]
Hi Prajwala1921,

I'm not sure if this is what you are looking for, but there are ways to calculate Weibull parameters (dissolution shape and time) from experimental dissolution profiles, for example [described in detail by Langenbucher](http://www.ncbi.nlm.nih.gov/pubmed/4146531). Note that the time interval here represents 63.2% of the dissolved dose, and thus does not match the t50 input value in PK-Sim, but it should at least give a good approximation. Hope this helps you.

Best regards, 
Simeon

**Reply by Prajwala1921** [reply]
Thank you so much @SRuedesh. This helps actually, I will try applying this method of linearisation. I have to check for the predictability of the model and whether it translates the minute differences between 2 formulations.

**Comment by AndreDlm** [comment]
Hi @Prajwala1921,

in addition to Simeon's answer you could consider modeling the dissolution profile in the in vitro setting using MoBi or R and then feed the results into your whole-body PBPK model in PK-Sim. 

For example, if you intend using the particle dissolution model, you may use the workflow described [here](https://github.com/Open-Systems-Pharmacology/IVIVC-with-particle-dissolution-module-in-OSP) where you first fit relevant parameters of the dissolution function to observed in vitro release kinetics and then push the parameterization of the dissolution function into the PBPK model in PK-Sim. 

Some time ago, I have started implementing a 2-compartment dissolution model with the Weibull function in MoBi. I have attached the file including hypothetical in vitro dissolution data here: 
[InVitroDissolution_Weibull.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/8797340/InVitroDissolution_Weibull.zip)
The model still has a few things that could be improved to make it more easy (e.g. the fraction dissolved can be >1, the implementation of the time lag and the dose is a bit cumbersome), but it's running. Feel free to use and improve it.

André

**Reply by Prajwala1921** [reply]
Hello @AndreDlm,
Thanks for the response. This is a very interesting thing to do, I will definitely follow this, I think this method suits the profile that possesses complex kinetics. Thank you again for sharing more in detail.

**Comment by fredsevmartins** [comment]
@Prajwala1921  Also you can use an app called DDsolver, it is free and you can model your formulation release there and plugin into PKSIM.

Here is the paper where you can get more information. doi: [10.1208/s12248-010-9185-1](https://doi.org/10.1208%2Fs12248-010-9185-1)

**Comment by finattig** [comment]
Is there any way to change the Weibull equation implemented in the software (e.g., to use only the a, b parameters, or incorporate Fmax) to input dissolution data when creating a new formulation?

**Reply by AndreDlm** [reply]
That's not possible in PK-Sim, but in MoBi you can change the dissolution equation for any (solid) formulation in the `Events` Building Block and introduce, e.g. a new variable for Fmax

**Reply by finattig** [reply]
Thank you

Regards
